controllers/openshift_controller.go

```diff
@@ -307,27 +307,27 @@ WantedBy=multi-user.target
 	}
 
 	file := ignTypes.File{}
-	c := ignTypes.FileContents{}
+	c := file.Contents
 
 	dropinConf, err := generateDropinConfig(r.kataConfig.Status.RuntimeClass)
 	if err != nil {
 		return nil, err
 	}
 
-	c.Source = "data:text/plain;charset=utf-8;base64," + dropinConf
+	dropinFile := "data:text/plain;charset=utf-8;base64," + dropinConf
+	c.Source = &dropinFile
```

Comment:
can we simply use `file.Contents.Source = &dropinFile` and avoid using `c` ?

Reply:
yes, we can! Thanks @bpradipt !